### ISSUE TITLE
[question] How to get available_datasets ?

### ISSUE THREAD

**Issue by alienzj** [bactopia/bactopia#337]
```bash
(bactopia) [jiezhu@hpc2021 bactopia]$ bactopia datasets --available_datasets
Traceback (most recent call last):
  File "/home/jiezhu/.conda/envs/bactopia/bin/bactopia-datasets.py", line 1018, in <module>
    available_datasets(PUBMLST)
TypeError: available_datasets() missing 1 required positional argument: 'pubmlst'
```
Dear @rpetit3  How to get available_datasets ?
Thanks!

**Comment by alienzj** [comment]
```
(bactopia) [jiezhu@hpc2021 bactopia]$ bactopia datasets --available_datasets --help                                                                                  
usage: bactopia datasets [-h] [--outdir STR] [--species STR] [--skip_mlst] [--skip_prokka] [--include_genus]                                                         
                         [--assembly_level {all,complete,chromosome,scaffold,contig,complete,chromosome,chromosome,complete}] [--limit INT] [--accessions STR]       
                         [--identity FLOAT] [--overlap FLOAT] [--max_memory INT] [--fast_cluster] [--skip_minmer] [--skip_amr] [--prodigal_tf STR]                   
                         [--reference STR] [--mapping STR] [--genes STR] [--proteins STR] [--primers STR] [--force_optional] [--cpus INT] [--clear_cache] [--force]  
                         [--force_mlst] [--force_prokka] [--force_minmer] [--force_amr] [--keep_files] [--available_datasets] [--available_species] [--depends]      
                         [--skip_ssl_check] [--version] [--verbose] [--silent]                                                                                       
                         PUBMLST                                                                                                                                     
                                                                                                                                                                     
bactopia datasets (v2.1.0) - Setup public datasets for Bactopia                                                                                                      
                                                                                                                                                                     
positional arguments:                                                                                                                                                
  PUBMLST               Bactopia config file with PubMLST schema mappings.                                                                                           
                                                                                                                                                                     
optional arguments:                                                                                                                                                  
  -h, --help            show this help message and exit                                                                                                              
  --outdir STR          Directory to write output. (Default ./datasets)                                                                                              
                                                                                                                                                                     
Bacterial Species:                                                                                                                                                   
  --species STR         Download available MLST schemas and completed genomes for a given species or a list of species in a text file.                               
  --skip_mlst           Skip setup of MLST schemas for each species                                                                                                  
                                                                                                                                                                     
Custom Prokka Protein FASTA:                                                                                                                                         
  --skip_prokka         Skip creation of a Prokka formatted fasta for each species                                                                                   
  --include_genus       Include all genus members in the Prokka proteins FASTA                                                                                       
  --assembly_level {all,complete,chromosome,scaffold,contig,complete,chromosome,chromosome,complete}                                                                 
                        Assembly levels of genomes to download (Default: complete).                                                                                  
  --limit INT           If available completed genomes exceeds a given limit, a random subsample will be taken. (Default 1000)                                       
  --accessions STR      A list of RefSeq accessions to download.                                                                                                     
  --identity FLOAT      CD-HIT (-c) sequence identity threshold. (Default: 0.9)                                                                                      
  --overlap FLOAT       CD-HIT (-s) length difference cutoff. (Default: 0.8)                                                                                         
  --max_memory INT      CD-HIT (-M) memory limit (in MB). (Default: unlimited                                                                                        
  --fast_cluster        Use CD-HIT's (-g 0) fast clustering algorithm, instead of the accurate but slow algorithm.
Minmer Datasets:
  --skip_minmer         Skip download of pre-computed minmer datasets (mash, sourmash)

Antimicrobial Resistance Datasets:
  --skip_amr            Skip download of antimicrobial resistance databases (e.g. AMRFinder+)

Optional User Provided Datasets:
  --prodigal_tf STR     A pre-built Prodigal training file to add to the species annotation folder. Requires a single species (--species) and will replace existing
                        training files.
  --reference STR       A reference genome (FASTA/GenBank (preferred)) file or directory to be added to the optional folder for variant calling. Requires a single
                        species (--species).
  --mapping STR         A reference sequence (FASTA) file or directory to be added to the optional folder for mapping. Requires a single species (--species).
  --genes STR           A gene sequence (FASTA) file or directory to be added to the optional folder for BLAST. Requires a single species (--species).
  --proteins STR        A protein sequence (FASTA) file or directory to be added to the optional folder for BLAST. Requires a single species (--species).
  --primers STR         A primer sequence (FASTA) file or directory to be added to the optional folder for BLAST. Requires a single species (--species).
  --force_optional      Overwrite any existing files in the optional folders

Custom Options:
  --cpus INT            Number of cpus to use. (Default: 1)
  --clear_cache         Remove any existing cache.
  --force               Forcibly overwrite existing datasets.
  --force_mlst          Forcibly overwrite existing MLST datasets.
  --force_prokka        Forcibly overwrite existing Prokka datasets.
  --force_minmer        Forcibly overwrite existing minmer datasets.
  --force_amr           Forcibly overwrite existing antimicrobial resistance datasets.
  --keep_files          Keep all downloaded and intermediate files.
  --available_datasets  List MLST schemas available for setup.
  --available_species   List species availble from current datasets.
  --depends             Verify dependencies are installed.

Adjust Verbosity:
  --skip_ssl_check      wget will run with --no-check-certificate to skip SSL checks
  --version             show program's version number and exit
  --verbose             Print debug related text.
  --silent              Only critical errors will be printed.

example usage:
  bactopia datasets
  bactopia datasets --species 'Staphylococcus aureus' --include_genus
```

**Comment by alienzj** [comment]
I installed bactopia v2.1.0 using mamba. From the above command line output, I can't see parameter `--ariab` which have been mentioned in the bactopia documentations.

**Comment by rpetit3** [comment]
Hi @alienzj 

This is likely dues to me removing Ariba to a Bactopia Tool in v2.1.0.

I'll get this fixed and update you soon,!

Robert

**Comment by rpetit3** [comment]
Ok, this is now fix (https://github.com/bactopia/bactopia/commit/798c69e458ecf04ffed8fa34dddaf816250e55b0) and is available in my dev build 

```
mamba create -n bactopia-dev -c rpetit3 -c conda-forge -c bioconda bactopia
conda activate bactopia-dev
```

Thank you for reporting this!

**Comment by alienzj** [comment]
Great ! Thanks Robert !

**Comment by rpetit3** [comment]
Hi @alienzj,

I've released [v2.1.1](https://github.com/bactopia/bactopia/releases/tag/v2.1.1) which includes a fix for this issue. If it persists, please feel free to reopen.

Cheers,
Robert
